packages/build-utils/src/prerender.ts

```diff
@@ -37,7 +37,9 @@ export class Prerender {
     this.expiration = expiration;
 
     this.lambda = lambda;
-    this.lambda.operationType = this.lambda.operationType || 'ISR';
+    if (this.lambda) {
```

Comment:
Can `lambda` be undefined/null?

Reply:
I saw an error saying so, but I now realize that was is a less-typed test passing it in. I'll update this.

Reply:
I removed the guard and fixed the test that caused this issue: [`e50c3ac` (#8887)](https://github.com/vercel/vercel/pull/8887/commits/e50c3acf74dd6ddd8d69ab5b47aeb52527102a8f)